modules/agent-install-sample-config-bond-sriov.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-with-agent-based-installer/preparing-to-install-with-agent-based-installer.adoc
+
+:_content-type: REFERENCE
+[id="agent-install-sample-config-bond-sriov_{context}"]
+= Example: Bonds and SR-IOV dual-nic node network configuration
+
+[IMPORTANT]
+====
+Support for Day 1 operations associated with enabling NIC partitioning for SR-IOV devices is a Technology Preview feature. Technology Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete. Red Hat does not recommend using them in production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/.
+====
+
+The following `agent-config.yaml` file is an example of a manifest for dual port NIC with a bond and SR-IOV interfaces.
+
+[source,yaml]
+----
+apiVersion: v1alpha1
+kind: AgentConfig
+rendezvousIP: 10.10.10.14
+hosts:
+ - hostname: worker-1
+    interfaces:
+     - name: eno1
+       macAddress: 0c:42:a1:55:f3:06
+     - name: eno2
+       macAddress: 0c:42:a1:55:f3:07
+    networkConfig: <1>
+      interfaces: <2>
+        - name: eno1     <3>
+          type: ethernet <4>
+          state: up
+          mac-address: 0c:42:a1:55:f3:06
+          ipv4:
+            enabled: true
+            dhcp: false  <5>
+          ethernet:
+            sr-iov:
+              total-vfs: 2 <6>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:0
+          type: ethernet
+          state: up  <7>
+          ipv4:
+            enabled: false <8>
+          ipv6:
+            enabled: false
+        - name: sriov:eno1:1
+          type: ethernet
+          state: down
+          min-tx-rate: 100 <9>
+          max-tx-rate: 200 <10>
```

Comment:
I suppose the qos configuration should be moved to the eno1:0 and eno2:0 interfaces since they are bond slaves.

Reply:
Hi @evgenLevin I moved under bond0 I hope that is the correct place. I need to update the other places this is used.